tests/Doctrine/Tests/DBAL/Functional/Schema/SqliteSchemaManagerTest.php

```diff
@@ -257,4 +257,51 @@ public function testOnlyOwnCommentIsParsed(): void
             ->getColumn('col1')
             ->getComment());
     }
+
+    public function testUnnamedForeignKeyConstraintHandling(): void
+    {
+        $sql = <<<SQL
+CREATE TABLE "unnamedfk_referenced1" (
```

Comment:
Namespacing table names doesn't yield enough test isolation. If the test is run against a persistent database, it will be impossible to run it twice. Use `DROP TABLE IF EXISTS`.